### ISSUE TITLE
Incorrect assumption of 'alt' text and SVGs

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/Icons#173]
## Describe the bug

Previous work updated the use of `title` and `desc` content within an SVG. There was also an assumption that an empty `title` or `desc` element would bypass the SVG image in a screen reader. 

While researching, it was also learned that all `alt` text should contain punctuation so that the screen reader can read back information correctly.

## Suggested correction 

- The data.json should be updated to move content from `title` to `desc`
- SVG generator needs to add `aria-labeledby` to the `svg` element and an ID to the `title` and `desc` elements.
- If there is no `title` or `desc` content, the icon should be hidden from screen readers

## Verify the version of @alaskaairux/icons you have installed

**The current version is:**
[![See it on NPM!](https://img.shields.io/npm/v/@alaskaairux/icons?style=for-the-badge&color=orange)](https://www.npmjs.com/package/@alaskaairux/icons)

My project has @alaskaairux/icons v@current installed

## To Reproduce

1. Go to https://auro.alaskaair.com/icons
1. Turn on a screen reader
1. Allow the screen reader to read back an icon that does not have a `title` or `desc` element with content
1. Notice that the SVG reports back as `img` with no other context. 

Are you able to reproduce this issue on the [Auro doc site](https://auro.alaskaair.com/)?

- [x] YES
- [ ] NO

## Expected behavior

- Icons are required to have `desc` content or should be hidden from screen readers
- Icons that require additional context support with a hover and tooltip require both `title` and `desc` content

## Exit criteria 

This issue will be considered complete once icons:

1. If `title` and/or `desc` content is available, the associated  `aria-labeledby` attribute will be added to the `svg` element
1. If no `title` or `desc` is associated with the icon, the associated  `labeledby` attribute will not be added to the `svg` element.
1. If both the `title` and `desc` content do not exist for an icon the `aria-hidden="true"` attribute will be added to the `svg` element and the `desc` elements will not be added to the `svg` element.
1. The `desc` content will contain proper punctuation
1. Screen reader alt text will be updated to the `desc` element versus the `title` element
1. Accessibility documentation will be added to the repo
